x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -202,6 +208,10 @@ export function ServiceInventory() {
     ...preloadedServices,
   ].some((item) => 'healthStatus' in item);
 
+  const displayMaxGroupReachedCallout = mainStatisticsItems.some(
+    (item) => item.serviceName === '_other'
+  );
```

Comment:
This is not the right check to do for the given warning. The warning message says:

> The number of unique service names limit was exceeded. Try narrowing down your results using the search bar.

However, the warning will be shown if a bucket called `_other` exists, which means that the number of unique transaction groups exceeded the APM Server limit. 

Narrowing down the results won't change this since APM Server already grouped data into `_other`.

What I was saying previously, is that you need a warning message when exceeding the kibana limit (10.000). This can happen if the Kibana limit is lower than the APM Server limit. 

You can check if `sum_other_doc_count > 0` to see if any buckets (transaction groups) were left out: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#terms-agg-doc-count-error

Reply:
We may still want to display a warning like this when APM Server limit is exceeded. You should just change the message to say the same (or something similar) as to what the tooltip (on the "_other") says